playerbot/strategy/triggers/GenericTriggers.h

```diff
@@ -928,6 +928,13 @@ namespace ai
         virtual bool IsActive() override;
     };
 
+    class UseRingTrigger : public BoostTrigger

```

Comment:
No need to inherit from BoostTrigger if nothing gets reused from that class. Better inherit from Trigger